### ISSUE TITLE
Unhandled error ApiError: Non-existent body parameter "metaData.order_id" for this API call.

### ISSUE THREAD

**Issue by Jackilion** [mollie/mollie-api-node#212]
I want to get started with the payment API. But when I run the example code displayed here:
https://docs.mollie.com/reference/v2/payments-api/create-payment

I get the Error

```
Unhandled error ApiError: Non-existent body parameter "metaData.order_id" for this API call.
    at Function.createFromResponse (/workspace/node_modules/@mollie/api-client/dist/mollie.cjs.js:247:12)
    at /workspace/node_modules/@mollie/api-client/dist/mollie.cjs.js:406:26
    at Generator.throw (<anonymous>)
    at asyncGeneratorStep (/workspace/node_modules/@mollie/api-client/dist/mollie.cjs.js:59:24)
    at _throw (/workspace/node_modules/@mollie/api-client/dist/mollie.cjs.js:85:9)
    at processTicksAndRejections (internal/process/task_queues.js:97:5) {
  title: 'Unprocessable Entity',
  status: 422,
  field: 'metaData.order_id',
  links: {
    documentation: {
      href: 'https://docs.mollie.com/guides/handling-errors',
      type: 'text/html'
    }
  }
} 
```

This is my code:

```
const payment = await mollieClient.payments.create({
    amount: {
    currency: "EUR",
    value: "10.00"
    },
    description: "Order test",
    redirectUrl: "xxx",
    webhookUrl: "xxx",
    metadata: {
        order_id: "123"
    }
});
```
I also tried providing a "metaData" argument, as it is called in the error, but I get the same error.

Any help is appreciated.

**Comment by hobadams** [comment]
@Jackilion - Have you tried `orderId: "123"`?

I've not tested this but my guess is this is your issue.

**Comment by Jackilion** [comment]
Thank you for the fast answer.

```
metadata: {
    orderId: "123"
}
```

did work, as you suspected.
Don't forget to change the example code on your get started docs

**Comment by hobadams** [comment]
@Jackilion - Can you provide a link to the doc? I don't work for Mollie (I just saw your comment because i'm following the repo :-) ).

**Comment by Jackilion** [comment]
@hobadams 
Oh, well thank you for the help nevertheless!
You can find the code from my original post here:
https://docs.mollie.com/reference/v2/payments-api/create-payment
just scroll to the bottom and click on Node.js.

**Comment by Pimm** [comment]
Hi, thanks for the issue.

You as a developer can decide what's in the metadata, as well as whether there is an `order_id`, an `orderId`, or an `orderID` (or frankly: none of the above or even all of them).

To be very sure, I copied the code you posted and tested it myself (only changing the URLs). And I'm sorry to say this, but it works for me.

![Screenshot of test payment](https://user-images.githubusercontent.com/78094/124480738-df443c00-dda7-11eb-8321-ff4e242adee0.png)

Could you please double-check that this was in fact the exact code that caused this error?

**Comment by Jackilion** [comment]
Thank you for the answer.
This was in fact the code I used. I just copied it directly out of my IDE, and x-ed out the links, that's it.
However, I can't replicate the bug myself now.
I'm using Firebase and their error logging is sometimes a little weird, it could be possible that they were displaying an old error inside the logs. But then I don't know why it worked today, I only changed "order_id" to "orderId", deployed it and ran my code again and it worked.

Feel free to close this, this seems to be on me somehow.
